### ISSUE TITLE
Plugin overrides color settings from StatusBarPlugin

### ISSUE THREAD

**Issue by i8ramin** [j-mcnally/cordova-statusTap#11]
When used together with the https://github.com/phonegap-build/StatusBarPlugin plugin, the color settings are lost and status bar changes back to default style.

Haven't had a chance to dig deep into the code to figure out why, but will update this ticket once I find it.

**Comment by madrobby** [comment]
Same problem here. Statusbar text is white at first for me, then changes to black after a few seconds.

**Comment by j-mcnally** [comment]
Is this an ios7 thing admittedly i have not tested on ios 7

**Comment by madrobby** [comment]
FWIW, it works correctly when I set `UIViewControllerBasedStatusBarAppearance` to NO in the `Info.plist` file of the project. Maybe the overlay needs to inherit the statusbar settings from the topmost view controller.

**Comment by j-mcnally** [comment]
Yes this makes sense, i think the box is draw as white or black but then
set as transparent so it would be surprising that its keying off the wrong
color.

**Comment by demerzel3** [comment]
Is there a fix to this? I'm trying to use this plugin alongside the StatusBar plug-in and have similar issues: with UIViewControllerBasedStatusBarAppearance set to YES the tap listener works, but the statusbar color/visibility is broken. With UIViewControllerBasedStatusBarAppearance set to NO the visibility works fine but no tap event is fired.

**Comment by jakecraige** [comment]
`UIViewControllerBasedStatusBarAppearance` to `NO` in `Info.plist` is working for me. Thanks @madrobby 

Just upgraded to Cordova 3.4.1, @demerzel3 perhaps try upgrading if you haven't and see if that fixes it.

**Comment by hodeyp** [comment]
Any update on this issue? The `UIViewControllerBasedStatusBarAppearance` change does not work for me as it makes the status bar disappear.

**Comment by gregavola** [comment]
I'm also having this problem - any help would be great!

**Comment by orenagiv** [comment]
Add this - and it will change the text color of the status-bar to white:

```
<key>UIStatusBarStyle</key>
<string>UIStatusBarStyleLightContent</string>
```

Make sure you also have the following in your plist:

```
<key>UIStatusBarHidden</key>
<true/>
<key>UIViewControllerBasedStatusBarAppearance</key>
<false/>
```
